app/src/main/res/values/strings.xml

```diff
@@ -703,4 +703,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_recycling_type_plastic_bottles">Plastic bottles only</string>
     <string name="quest_recycling_type_plastic_packaging">Plastic packaging only</string>
     <string name="quest_recycling_type_scrap_metal">Scrap metal</string>
+    <string name="quest_recycling_type_any_glass">Any glass</string>
+    <string name="quest_recycling_glass_title">Can only glass bottles be recycled here, or any type of glass?</string>
```

Comment:
I would explicitly include jars in question and specifically describe weird types of glass - window glass, tempered glass, pyrex, light bulb glass, glassware and so on.

Without that I would happily answer "yes, all kinds of glass" and be sure that I am correct.

I even opened https://github.com/osm-fr/osmose-backend/issues/549 thinking that validator is wrong.

Maybe ask that in "are you sure dialog"? Or have additional text below like wheelchair quest has?

Reply:
I can make additional PR adding this.

Reply:
The question should not get too long and the button "Any glass" must also stay very short.  So I think an additional text like in the wheelchair quest would be best. I'd gladly accept a PR